javascript/javascript1/week3/notePad.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const notes = []
+function saveNote(content, id) {
+    const obj = {content, id}
```

Comment:
It is nice to write {content, id} instead of {content: content, id: id}. It calls restructuring you can read more about it here: 
https://dmitripavlutin.com/javascript-object-destructuring/